Library/Formula/pacapt.rb

```diff
@@ -0,0 +1,19 @@
+# Pacapt is a wrapper for all major package managers. It allows
+# a user to use only one pacman-likes syntax while working with
+# different package managers.
+
+class Pacapt < Formula
+
+  homepage "https://github.com/letalvoj/pacapt" 
+  url "https://github.com/icy/pacapt.git", :using => :git, :tag => "v2.0.2"
+
+  def install
+    system "make", "install", "BINDIR=#{prefix}"
+    bin.install_symlink "#{prefix}/pacapt" => "pacapt"
+  end
+
+  test do
+    system bin/"pacapt", "-h"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!